package.json

```diff
@@ -67,17 +66,18 @@
     "weaktuplemap": "^1.0.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^8.2.1",
+    "@babel/cli": "7.0.0",
+    "@babel/core": "7.0.0",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
For shared knowledge: this "babel-core" bridge is currently necessary for Jest. I don't know why but I suspect this is related to `babel-jest` and its dependencies. Without this the tests are failing with an error about babel version.